x/epochs/types/hooks.go

```diff
@@ -0,0 +1,33 @@
+package types
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+type EpochHooks interface {
+	// the first block whose timestamp is after the duration is counted as the end of the epoch
+	OnEpochEnd(ctx sdk.Context, epochIdentifier string, epochNumber int64)
```

Comment:
OnEpochEnd => AfterEpochEnd 
OnEpochStart => BeforeEpochStart
This would be better for common hook name styles.

Reply:
Will make changes